app/src/views/ResultPage.vue

```diff
@@ -164,7 +164,8 @@
                       {{post.title}}
                     </v-list-item-title>
                     <v-list-item-subtitle :set="distance = postDistance(post)">
-                      <strong>{{post.location}} <em v-if="distance">(in {{distance}})</em></strong> &mdash; {{post.task}}
+                      <strong>{{post.location}} <em v-if="distance">(in {{distance}})</em></strong> &mdash;
+                       <label class="hide-copyright" v-html="post.task"/>
```

Comment:
The <label> element is used to associate an label with an input element.
Since there is no associated input for this label please use a <span> or <p> tag